Tests/Shared.Specs/NumericAssertionSpecs.cs

```diff
@@ -876,6 +876,25 @@ public void When_asserting_a_byte_value_it_should_treat_is_any_numeric_value()
             act.Should().NotThrow();
         }
 
+        [Fact]
+        public void When_asserting_a_sbyte_value_it_should_treat_is_any_numeric_value()
```

Comment:
In general, I'd like to avoid these repetitive `When_asserting`. It adds superfluous noise.